stories/forms/SampleForm.stories.tsx

```diff
@@ -5,6 +5,9 @@ import TextInput from '@leafygreen-ui/text-input';
 
 export default {
   title: 'Form/Sample Forms',
+  parameters: {
```

Comment:
do we even need this file?

Reply:
Not sure. Sean created it. I think it's a wip